gst/nnstreamer/tensor_query/tensor_query_serversink.c

```diff
@@ -217,17 +217,13 @@ static gboolean
 gst_tensor_query_serversink_set_caps (GstBaseSink * bsink, GstCaps * caps)
 {
   GstTensorQueryServerSink *sink = GST_TENSOR_QUERY_SERVERSINK (bsink);
-  gchar *caps_str, *prev_caps_str, *new_caps_str;
+  gchar *caps_str, *new_caps_str;
 
   caps_str = gst_caps_to_string (caps);
 
-  nns_edge_get_info (sink->edge_h, "CAPS", &prev_caps_str);
-  if (!prev_caps_str)
-    prev_caps_str = g_strdup ("");
-  new_caps_str = g_strdup_printf ("%s@query_server_sink_caps@%s",
-      prev_caps_str, caps_str);
-  nns_edge_set_info (sink->edge_h, "CAPS", new_caps_str);
-  g_free (prev_caps_str);
+  new_caps_str = g_strdup_printf ("@query_server_sink_caps@%s", caps_str);
+  gst_tensor_query_server_set_caps (sink->server_h, new_caps_str);
```

Comment:
gst_tensor_query_server_set_caps return immediately if sink->server_h is NULL.
in this case, 
gst_tensor_query_serversink_set_caps is return true.
I wonder if it should return  FALSE or not.